### PR TITLE
Migrated authenticated-requests to null safety

### PR DESCRIPTION
**Pull request by miquelbeltran**
Added comments to the PR

cc. @sfshaza2 @RedBrogdon

### CODE REVIEW

File: src/docs/cookbook/networking/authenticated-requests.md [hunk 1]
```diff
@@ -20,22 +22,22 @@ convenient way to add headers to your requests.
...
-<!-- skip -->
+<?code-excerpt "lib/main.dart (get)"?>
 ```dart
-Future<http.Response> fetchAlbum() {
-  return http.get(
-    Uri.parse('https://jsonplaceholder.typicode.com/albums/1'),
-    // Send authorization headers to the backend.
-    headers: {HttpHeaders.authorizationHeader: "Basic your_api_token_here"},
-  );
-}
```
Comment (miquelbeltran) [inline src/docs/cookbook/networking/authenticated-requests.md:31]: I simplified a bit the example by removing the function declaration surrounding it

**Comment by sfshaza2** [comment]
I'm ok with the simpler code. @domesticmouse?  (I believe he's on vacation for the rest of the week...)

**Review by domesticmouse — APPROVED** [review]

**Comment by sfshaza2** [comment]
DESPITE being on VACATION, @domesticmouse is still working! Thanks for the quick review, @domesticmouse. Shame on you for working.  :D

**Comment by domesticmouse** [comment]
Nothing better to do while waiting for the truck to deliver my furniture at the new house 😂

**Comment by miquelbeltran** [comment]
Thanks for the review and congrats on the new place! :)

**Comment by domesticmouse** [comment]
Thanks @miquelbeltran =)

**Comment by sfshaza2** [comment]
It's pretty cool that BOTH of you have new houses, @miquelbeltran and @domesticmouse!

**Pull request merged**